lib/config/config.go

```diff
@@ -112,14 +112,14 @@ func (c *Configuration) Save() *errco.Error {
 
 // loadDefault loads config file to config variable
 func (c *Configuration) loadDefault() *errco.Error {
-	// get msh executable path
-	mshPath, err := os.Executable()
+	// get working directory
+	cwdPath, err := os.Getwd()
 	if err != nil {
 		return errco.NewErr(errco.ERROR_CONFIG_LOAD, errco.LVL_1, "loadDefault", err.Error())
 	}
 
 	// read config file
```

Comment:
since I know many users will have silly problems because of msh reading a config in a different folder (for whatever reason), can you add a log line with `"reading config file at: " + cwdPath` (`LVL_1`)

Reply:
@gekigek99 Added.

```[minecraft@yggdrasil ~]$ msh
╔═════════════════════════════════════════════╗
║  _ __ ___  ___| |__                         ║
║ | '_ ` _ \/ __| '_ \                        ║
║ | | | | | \__ \ | | | v2.4.6                ║
║ |_| |_| |_|___/_| |_| -------               ║
║ Copyright (C) 2019-2022 gekigek99           ║
║ github: https://github.com/gekigek99        ║
║ remember to give a star to this repository! ║
╚═════════════════════════════════════════════╝
2022/06/16 12:56:47 [info  ≡   ] checking OS support...
2022/06/16 12:56:47 [info  ≡   ] loading config...
2022/06/16 12:56:47 [info  ≡≡≡ ] reading config file at: /home/minecraft/msh-config.json
2022/06/16 12:56:47 [info  ≡≡≡≡] setting log level to: 2
2022/06/16 12:56:47 [info  ≡≡≡ ] eula.txt exist and is set to true
2022/06/16 12:56:47 [info  ≡≡≡ ] listening for new clients to connect on 0.0.0.0:25555 ...
```